components/web/aget/patches/01_Makefile.Solaris.patch

```diff
@@ -0,0 +1,14 @@
+--- aget-0.4.1/Makefile.Solaris		2016-12-21 22:44:23.411350275 -0200
```

Comment:
Please, patch Makefile to use $(CC) and $(CFLAGS), $(LDFLAGS), passed by oi-userland build system (and check that they are actually passed).  Something like 
```
CC ?= gcc
CFLAGS  = -g -W -DSOLARIS -D_LARGEFILE64_SOURCE -D_FILE_OFFSET_BITS=64 $(MYCFLAGS) 
```
and exporting CC and MYCFLAGS=$(CFLAGS) in build environment should work.
Note, that if you want to deliver only 64-bit version of aget, you don't need largefile compilation environment.